src/resources/views/login.blade.php

```diff
@@ -0,0 +1,48 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    @vite(['resources/css/app.css', 'resources/js/app.js', 'resources/scss/app.scss'])
+    <title>Login</title>
```

Comment:
I think this should be on a separate file like app.blade.php. 

app.blade.php is kinda like "index" where you import sections, layouts, pages etc using `@include`, `@yield` etc. So if login is a page, it should be inside the folder views/page, then inside the app.blade.php you only have to call the login page.

Reference:
https://www.geeksforgeeks.org/laravel-blade-templates-inheritance/

Reply:
I have made changes Sir. Thank you!